docker/aladdin-demo.Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM alpine:latest
+
+# install python and pip with apk package manager
+RUN apk -Uuv add python py-pip
+
+# uwsgi in particular requires a lot of packages to install, delete them afterwards
+RUN apk add --no-cache \
+        gettext \
+        python3 \
+        build-base \
+        linux-headers \
+        python3-dev
+
+# copies requirements.txt to the docker container
+ADD requirements.txt requirements.txt
+
+# Install requirements
+RUN pip3 install --no-cache-dir -r requirements.txt
+
+# clean up environment by deleting extra packages
+RUN apk del \
+        build-base \
+        linux-headers \
+        python3-dev
+
+# specify the directory that CMD executes from
+WORKDIR /home/aladdin-demo
+
+# copy over the directory into docker container with given path
+COPY . /home/aladdin-demo
```

Comment:
Would be great to put all the application code into some folder like 'server' or 'app', and then only copy that directory - that way we wouldn't copy over things like Dockerfile/lamp.json etc, which aren't necessary in running the app.